advocacy_docs/partner_docs/VisionXGuide/05-UsingVisionx.mdx

```diff
@@ -0,0 +1,485 @@
+</p>
+
+#### Modify or add any Java code using Eclipse
+
+1. Import the application in Eclipse by clicking shift+alt in the VisionX main view and clicking the icon next to the application.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration14.png">
+</p>
+
+2. Select an UI element in VisionX and click the purple Eclipse button.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration15.png">
+</p>
+
+3. Eclipse navigates to the relevant Java Code. 
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration16.png">
+</p>
+
+4. Change the “Order Information” label in Eclipse.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration17.png">
+</p>
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration18.png">
+</p>
+
+   If you change the code outside of VisionX in Eclipse, the changes are pushed back into the VisionX visual development environment in real time! This enables 100% unlimited app development for Citizen Developers / Business Users and Pro Developers.
+
+10. Open the screen in the VisionX Live Preview and change some of the data. The data will be updated in the Oracle Database.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration19.png">
+</p>
+
+### Oracle database to EDB Postgres Advanced Server migration
+
+In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended Server using the EDB's MTK migration tool.
+
+The migration steps in connection with an Oracle Forms migration are not different from a pure Oracle to EDB database migration. Therefore, we do not discuss this topic in detail here.  Links to MTK
+
+Refer to the [MTK documentation](https://www.enterprisedb.com/docs/migration_toolkit/latest/)
+
+### Switch database connection to EDB Postgres Advanced Server
+
+Switching from the Oracle database is a simple step in VisionX. In the migrated VisionX application, simply click on the VisionX Settings screen and choose the “Datasources” tab. Click “Edit” to change the database connection from Oracle to PostgresSQL.
+
+<p align="center">
+  <img  alt="Create a new Application" src="Images/FMBMigration20.png">
+</p>
+
+## Data model-based migration of an Oracle Forms / APEX application to an EDB Postgres Advanced Server / VisionX Low Code application
+
+This example shows how an Oracle Forms / APEX application is migrated to a VisionX low-code application using the existing data model. The first step is to migrate the application. In the second step, the Oracle database is migrated to EDB Postgres Advanced Server/EDB Postgres Extended using the usual tools such as EDBs MTK.
+
+Then the migrated application is connected to EDB Postgres Advanced Server/EDB Postgres Extended Server.
+
+### Data model based application migration
+
+Steps 1 to 5 are the same as for an FMB-based application migration.
```

Comment:
Change
Steps 1 to 5 are the same as for an FMB-based application migration.
to
Follow Steps 1 to 5 under section FMB-based application migration. and then go to step 1.

Update steps to start with 1, then 2, and so on.